managed/CounterStrikeSharp.API/Core/Model/CCheckTransmitInfo.cs

```diff
@@ -0,0 +1,49 @@
+﻿/*
+ *  This file is part of CounterStrikeSharp.
+ *  CounterStrikeSharp is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  CounterStrikeSharp is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with CounterStrikeSharp.  If not, see <https://www.gnu.org/licenses/>. *
+ */
+
+using System.Runtime.InteropServices;
+
+namespace CounterStrikeSharp.API.Core
+{
+    [StructLayout(LayoutKind.Sequential)]
+    public struct CCheckTransmitInfo
+    {
+        /// <summary>
+        /// Entity n is already marked for transmission
+        /// </summary>
+        public CFixedBitVecBase TransmitEntity;
```

Comment:
Opinions on naming this `TransmitEntities`?

Reply:
Is in not better, `TransmittedEntities`, as they are `marked already` per initial summary, so it becomes `TransmittedEntities.Remove(entityIndex)`?

Reply:
They are not transmitted yet! they are just marked for transmission, thats why I'm thinking about `TransmitEntities` (it sounds like 'list of entities for transmit')